inc/common/purge.php

```diff
@@ -141,6 +141,9 @@ function rocket_get_purge_urls( $post_id, $post ) {
 		}
 	}
 
+	// Remove entries with empty values in array.
+	$purge_urls = array_filter( $purge_urls );
```

Comment:
```suggestion
	$purge_urls = array_filter( $purge_urls );
```
```suggestion
	$purge_urls = array_filter( $purge_urls, 'is_string' );
```

This change will ensure we only preserve string values in the array